dev-docs/creating_bindings.md

```diff
@@ -13,3 +13,11 @@ Keeping similar names and purposes for functions can make it easier for your use
 ## Community
 
 When ready, make a [pull request](https://github.com/uber/h3/edit/master/docs/community/bindings.md) to add your binding to the H3 documentation website.
+
+## Documentation
+
+To be included in the H3 [API reference](https://uber.github.io/h3/#/documentation/api-reference/), your binding should:
+
+* Be reasonably up to date with uber/h3.
+* Include bindings for the functions in listed in the output of `make binding-functions`.
```

Comment:
We might caveat this, as even our own bindings don't necessarily implement bindings for everything (e.g. JS and Python won't implement `h3ToString` or `stringToH3`, and we've talked about not implementing `hexRange` and family). So maybe `all appropriate functions` or something?

Reply:
Added an example of `stringToH3` for languages that only support the string H3 index type. `hexRange` and `radsToDegs` would also be good examples.